### DISCUSSION TITLE
Does this app clash with fan curves defined in BIOS?

### DISCUSSION THREAD

**Discussion by kelltom** [Rem0o/FanControl.Releases#375] [Q&A]
I use an ASUS board and have a bunch of QFan curves set up. Will this app override these settings, or re-write them?

**Comment by Rem0o** [comment]
Override. As soon as your computer restarts, go to sleep, or the app closes, everything resets.

**Comment by Mandoscottie** [comment]
ive found best setup is PWM on all devices and set to full speed (turbo run slightly lower than 100%) in Asus Bios. that way until windows loads, and fan control starts, all fans run at 100% once in windows all control is via FC. 

ive added a mix curve for CPU fans (2x140s on a 280mm rad up  top) and case fans, while leaving the 3080tis fans controlled from GPU temp. The mix curve is used for all other case fans on CPU/GPU temps, ive found this works very effectively with the 3080ti boosting higher than ever thanks to the improved thermal management.

**Reply by liamauve** [reply]
all fans at 100% during boot is not "best setup"
